src/main/java/de/zalando/tip/zalenium/servlet/LiveNodeHtmlRenderer.java

```diff
@@ -0,0 +1,226 @@
+package de.zalando.tip.zalenium.servlet;
+
+import com.google.gson.JsonObject;
+import org.openqa.grid.common.SeleniumProtocol;
+import org.openqa.grid.common.exception.GridException;
+import org.openqa.grid.internal.RemoteProxy;
+import org.openqa.grid.internal.TestSession;
+import org.openqa.grid.internal.TestSlot;
+import org.openqa.grid.internal.utils.HtmlRenderer;
+import org.openqa.grid.web.servlet.beta.MiniCapability;
+import org.openqa.grid.web.servlet.beta.SlotsLines;
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.remote.CapabilityType;
+
+import java.util.Map;
+
+public class LiveNodeHtmlRenderer implements HtmlRenderer {
+    private RemoteProxy proxy;
+
+    public LiveNodeHtmlRenderer(RemoteProxy proxy) {
+        this.proxy = proxy;
+    }
+
+    public String renderSummary() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("<div class='proxy'>");
+        builder.append("<p class='proxyname'>");
+        builder.append(proxy.getClass().getSimpleName());
+
+        builder.append(getHtmlNodeVersion());
+
+        String platform = getPlatform(proxy);
+
+        builder.append("<p class='proxyid'>id : ");
+        builder.append(proxy.getId());
+        builder.append(", OS : ").append(platform).append("</p>");
+
+        builder.append(nodeTabs());
+
+        builder.append("<div class='content'>");
+
+        builder.append(tabBrowsers());
+        builder.append(tabConfig());
+
+        builder.append("</div>");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "</div>" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)